packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -722,6 +724,7 @@ export abstract class SharedObject<
 	protected abstract summarizeCore(
 		serializer: IFluidSerializer,
 		telemetryContext?: ITelemetryContext,
+		incrementalContext?: IIncrementalContext,
```

Comment:
How are DDSes supposed to use this? This looks like an intermediate step to set up DDSes to use this. Without seeing how this will be used, its hard to review if this is fine or not.

Reply:
https://github.com/microsoft/FluidFramework/pull/14752, this is a base level issue, the tests that are failing here are caused by the testUtils change because the DDS doesn't exist. The test to look at is mapIncremental.spec.ts the last test is the incremental change.

Reply:
Created a test :)